src/utils/airGap.ts

```diff
@@ -0,0 +1,60 @@
+import bs58check from 'bs58check';
+import { URDecoder, UR, UREncoder } from '@ngraveio/bc-ur';
+import { IAccount, IAirgapAccountRaw } from '@/types';
+
+/**
+ * Type guard for TypeScript.
+ * Check if the account is an AirGap account and set the type accordingly.
+ */
+export function isAirgapAccount(
```

Comment:
In most places you use "AirGap" notation.

```suggestion
export function isAccountAirGap(
```